pkg/plugins/helm/parser.go

```diff
@@ -409,6 +409,10 @@ func (p *Parser) ParseCustomValues(conf []byte) (map[string]interface{}, string,
 	}
 	desc, _ := p.getStringFromValues(customVals, "Description")
 
+	if name == "" {
```

Comment:
Add this into CheckResource. 

Parse will be called when UpdateClusterEnv, so check should not be added in parse.